### PR TITLE
Feature/e2e tree view

### PR DESCRIPTION
**Pull request by tanyatrayanava**

### CODE REVIEW

File: tests/e2e/tests/critical-path/tree-view/tree-view.e2e.ts [hunk 1]
```diff
@@ -32,56 +35,57 @@ test
...
     });
 test
     .meta({ rte: rte.standalone })
     ('Verify that user can see DB is automatically scanned by 10K keys in the background, user can see the number of keys scanned and use the "Scan More" button to search per another 10000 keys', async t => {
         await t.click(browserPage.treeViewButton);
         //Verify the scanned value
-        await t.expect(browserPage.treeViewScannedValue.textContent).contains('Scanned 10 000', 'The database is automatically scanned by 10K keys');
+        await t.expect(browserPage.scannedValue.textContent).eql('10 000', 'The database is automatically scanned by 10K keys');
         //Verify that user can use the "Scan More" button to search per another 10000 keys
         await t.click(browserPage.scanMoreButton);
```
Comment (elenaNaboko) [inline tests/e2e/tests/critical-path/tree-view/tree-view.e2e.ts:47]: Cloud you please add a loop with clicking on Scan more button several times just to validate performance and FE still works

File: tests/e2e/tests/critical-path/tree-view/tree-view.e2e.ts [hunk 2]
```diff
@@ -32,56 +35,57 @@ test
...
-        await t.expect(browserPage.treeViewScannedValue.textContent).contains('Scanned 20 000', 'The database is automatically scanned by 10K keys');
+        await t.expect(browserPage.scannedValue.textContent).contains('20 000', 'The database is automatically scanned by 10K keys');
     });
 test
     .meta({ rte: rte.standalone })
-    ('Verify that user can see that “:” (colon) used as a default separator for namespaces', async t => {
+    ('Verify that user can see that “:” (colon) used as a default separator for namespaces and see the number of keys found per each namespace', async t => {
```
Comment (elenaNaboko) [inline tests/e2e/tests/critical-path/tree-view/tree-view.e2e.ts:52]: Maybe it would be better to move this test to regression suit?

**Review by elenaNaboko — CHANGES_REQUESTED** [review]

File: tests/e2e/tests/critical-path/tree-view/tree-view.e2e.ts (hunk 1 above)
Comment (tanyatrayanava) [reply tests/e2e/tests/critical-path/tree-view/tree-view.e2e.ts]: added

File: tests/e2e/tests/critical-path/tree-view/tree-view.e2e.ts (hunk 2 above)
Comment (tanyatrayanava) [reply tests/e2e/tests/critical-path/tree-view/tree-view.e2e.ts]: done

**Review by elenaNaboko — APPROVED** [review]

**Pull request merged**